openmp/runtime/src/include/ompx.h.var

```diff
@@ -9,6 +9,12 @@
 #ifndef __OMPX_H
 #define __OMPX_H
 
+#ifdef __AMDGCN_WAVEFRONT_SIZE
+#define __WARP_SIZE __AMDGCN_WAVEFRONT_SIZE
+#else
+#define __WARP_SIZE 32
```

Comment:
Why is 32 the default? 64 is the fallback if nothing is set AFAIK. You could also make this go to `(__builtin_amdgcn_wavefrontsize())`. The builtin is not great because it confounds constant optimizations, but in the case where it's not set at all it can be useful.

Reply:
For NVIDIA GPU it is 32. For AMDGPU, it is `__AMDGCN_WAVEFRONT_SIZE`.